### DISCUSSION TITLE
heating problem BTT SKR Mini E3 V3

### DISCUSSION THREAD

**Discussion by alsahime** [mriscoc/Ender3V2S1#940] [Q&A]
Unfortunately, I'm experiencing an error in the software. Can someone help me?

I have installed Mriscoc's firmware for Ender 3 V2 with BTT SKR MINI E3 V3, but when I try to heat it up, I receive a preheat error. The heating process is very slow and it gives this error before completion. I'm also encountering the same error during the PID test, and as a result, I am unable to print anything. I am not skilled in custom software development. Can anyone assist me with this?

**Comment by mriscoc** [comment]
Duplicate of https://github.com/mriscoc/Ender3V2S1/discussions/923#discussioncomment-6053640

**Reply by alsahime** [reply]
no there is still problem

**Reply by alsahime** [reply]
WHY U CLOSED THE CASE

**Reply by mriscoc** [reply]
Because the same question was posted in multiple places, so it is a duplicate. You must post only in one place in this Github.
